modules/ossm-security-mtls.adoc

```diff
@@ -15,17 +15,17 @@ By default, {ProductName} is set to permissive mode, where the sidecars in {Prod
 [id="ossm-security-enabling-strict-mtls_{context}"]
 == Enabling strict mTLS across the mesh
 
-If your workloads do not communicate with services outside your mesh and communication will not be interrupted by only accepting encrypted connections, you can enable mTLS across your mesh quickly. Set `spec.istio.global.mtls.enabled` to `true` in your `ServiceMeshControlPlane` resource. The operator creates the required resources.
+You can quickly enable mTLS across your mesh if your workloads do not communicate with outside services, and communication will not be interrupted by accepting only encrypted connections. Set `spec.security.mutualTLS.controlPlane.enable` to `true` in your `ServiceMeshControlPlane` resource. The operator creates the required resources.
 
 [source,yaml]
 ----
-apiVersion: maistra.io/v1
 kind: ServiceMeshControlPlane
 spec:
-  istio:
-    global:
-      mtls:
-        enabled: true
+  version: v2.0
+  security:
+      mutualTLS:
+        controlPlane: 
+          enable: true
```

Comment:
Indentation one level too deep. 

Also, `security.mutualTLS.controlPlane.enabled: true` should be `security.controlPlane.mtls: true`